ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -952,13 +952,13 @@ const triggerSet: TriggerSet<Data> = {
       id: 'R4S Left Roll',
       type: 'Ability',
       netRegex: { id: '95D3', source: 'Wicked Thunder', capture: false },
-      response: Responses.goLeft(),
+      response: Responses.goWest(),
```

Comment:
Oh this is for the platform breaking, so left is always west, and right is always east. Makes sense.